packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -98,12 +97,13 @@ export class Persona extends BaseComponent<IPersonaProps, IPersonaState> {
       onRenderOptionalText,
       imageShouldStartVisible
     } = this.props;
+    let size = this.props.size as PersonaSize;
```

Comment:
The type is inferred here. We typically don't specify the type when the compiler is able to figure it out (usually the case when doing an assignment with an initial value). Doing size: PersonaSize would still need a typecast anyway to get around strictNullChecks.